io/exportForGit.m

```diff
@@ -51,7 +51,14 @@
     RAVENver = fscanf(fid,'%s');
     fclose(fid);
 catch
-    RAVENver = 'unknown';
+    try
+        currentPath=pwd;
+        cd(RAVENpath);
+        RAVENver = git('log -n 1 --format=%H');
```

Comment:
How about making this consistent with the COBRA commit, i.e. with the word "commit" and a short hash? If you want I can implement it:
```
RAVEN_toolbox	d9698b2b6db268c63ee3386d5b784a544912549d
COBRA_toolbox	commit d7ce7d6
```

Reply:
Yes! I don't know why I got the longer hash, as I used the same command as for getting the Cobra hash? But if you now how to fix it, then please :)